test_integrations/phoenix_app/mix.exs

```diff
@@ -35,12 +35,11 @@ defmodule PhoenixApp.MixProject do
       {:nimble_options, "~> 1.0"},
       {:nimble_ownership, "~> 0.3.0 or ~> 1.0"},
 
-      {:phoenix, "~> 1.7.14"},
       {:postgrex, ">= 0.0.0"},
+      {:phoenix, "~> 1.7.14"},
       {:phoenix_html, "~> 4.1"},
+      {:phoenix_live_view, "~> 1.0", override: true},
```

Comment:
You don't need the override anymore:

```suggestion
      {:phoenix_live_view, "~> 1.0"},
```

Reply:
@whatyouhide seems like this broke CI :/

Reply:
Seems like a caching issue, purge the cache with the `gh` CLI and you should be good to go.